.github/workflows/sync-preprod-with-sanitised-prod-backup.yaml

```diff
@@ -0,0 +1,110 @@
+name: Sync PreProd with Sanitised Prod Backup
+
+on:
+  workflow_dispatch:
+    inputs:
+      backup-file:
+        description: Name of the prod backup file to sanitise and restore on preproduction (e.g., afqts_pd_2025-01-10.sql.gz)
+        required: true
+        type: string
+
+env:
+  SERVICE_NAME: apply-for-qts
+  SERVICE_SHORT: afqts
+  TF_VARS_PATH: terraform/application/config
+
+jobs:
+  sanitise-backup-and-restore-preproduction:
+    name: Sanitise Backup and Restore Preproduction
+    runs-on: ubuntu-latest
+    environment: preproduction
```

Comment:
Because it's a download of the prod backup first, the environment needs to be production.

Reply:
yes. The production env uses it's secrets to log into s189 prod and download the backup. GLOBAL_CONFIG sets the preprod config used by the restore, plus the workflow runs az login again using the _PREPRODUCTION azure secrets to access the s189 test subscription.